runtime/src/accounts_db.rs

```diff
@@ -6847,7 +6862,7 @@ impl AccountsDb {
         let hash = Hash::from_str(string).unwrap();
         let owner = Pubkey::from_str(string).unwrap();
         let lamports = 100_000_000;
-        let space = 0;
+        let space = self.filler_accounts_count * self.filler_accounts_size;
```

Comment:
This is the space _per account_, so only use the `filler_accounts_size` (not multiplied by `filler_accounts_count`).